.github/workflows/testing.yml

```diff
@@ -70,6 +70,9 @@ jobs:
       - name: Run tests
         run: poetry run pytest sycamore/tests/unit/
         working-directory: lib/sycamore
+      - name: Run more tests
+        run: poetry run python sycamore/tests/manual/test_fast_sycamore_import.py
+        working-directory: lib/sycamore
```

Comment:
heh heh this is gonna annoy some people... Maybe the error message in the tests should explain that you're not allowed to import ray in such a way that it gets imported with sycamore?

Reply:
Updated assertion error.  I didn't want to have to keep fixing the problem.